ccs/interviews.md

```diff
@@ -0,0 +1,269 @@
+# Interviews API Endpoints
+
+* [Get Interview](#get-interviewsinterviewid)
+* [Get Interviews](#get-interviews)
+* [Create Interview](#post-interview)
+* [Update Interview](#put-interviewsinterviewid)
+* [Delete Interview](#delete-interviewsinterviewid)
+
+---
+
+### `GET` `/interviews/:id`
```

Comment:
The line in line 3 of this file doesn't currently work.
The following update should make it work.
```suggestion
### `GET` `/interviews/:interviewId`
```